src/totp/totp.cpp

```diff
@@ -118,15 +136,16 @@ QString Totp::writeSettings(const QSharedPointer<Totp::Settings>& settings,
             urlstring.append("&encoder=").append(settings->encoder.name);
         }
         return urlstring;
-    }
-
-    // Semicolon output [step];[encoder]
-    if (!settings->encoder.shortName.isEmpty()) {
+    } else if (settings->keeOtp) {
+        // KeeOtp output
+        return QString("key=%1&size=%2&step=%3").arg(settings->key).arg(settings->digits).arg(settings->step);
```

Comment:
Shouldn't the key be URL-encoded, just in case someone puts something that's not plain Base64 in it?

Reply:
Done